qutebrowser/config/configdata.yml

```
@@ -1559,10 +1559,17 @@ spellcheck.languages:
 
 ## statusbar
 
-statusbar.hide:
-  type: Bool
-  default: false
-  desc: Hide the statusbar unless a message is shown.
+statusbar.show:
+  default: always
+  type:
+    name: String
+    valid_values:
+    - always: Always show the statusbar.
+    - never: Always hide the statusbar unless a message is shown.
+    - in_mode: >- 
```

```suggestion
    - in-mode: >- 
```

For consistency with other setting values.